update README and include updated tubemap (#148)
nf-core/seqinspector: README.md

| `QC`                | [`FastqScreen`](https://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/)  | Basic contamination detection               | [RNA, DNA]              | [N/A]               |
| `QC`                | [`SeqFu Stats`](https://github.com/telatin/seqfu2)                                 | Sequence statistics                         | [RNA, DNA]              | [N/A]               |
| `Indexing, Mapping` | [`Bwamem2`](https://github.com/bwa-mem2/bwa-mem2)                                  | Align reads to reference                    | [RNA, DNA]              | [N/A]               |
| `Indexing`          | [`SAMtools`](http://github.com/samtools)                                           | Index aligned BAM files, create FASTA index | [DNA]                   | [N/A]               |
| `QC`                | [`Picard`](https://broadinstitute.github.io/picard/picard-metric-definitions.html) | Collect multiple QC metrics                 | [RNA, DNA]              | [Bwamem2, SAMtools] |

> | `Reporting`         | [`MultiQC`](http://multiqc.info/)                                                  | Present QC for raw reads                    | [RNA, DNA, synthetic]   | [N/A]               |

Reviewer: ```suggestion
| `Reporting`         | [`MultiQC`](http://multiqc.info/)                                                  | Aggregate and visualise results                    | [RNA, DNA, synthetic]   | [N/A]               |
```
